k8s/prod/common/em/em-npa.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: em-npa
+  namespace: em
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.em-npa: glob:prod-*
+spec:
+  releaseName: em-npa
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: em-npa
+    version: 0.1.11
+  values:
+    anno:
+      replicas: 2
+      applicationInsightsInstrumentKey: "039faf8c-4682-432e-8fc5-f9735bb4b962"
```

Comment:
Same as above, u can get rid of ti from here by changing bootstrap.yaml

Reply:
you actually have it correct here : https://github.com/hmcts/rpa-native-pdf-annotator-app/blob/master/src/main/resources/bootstrap.yaml#L11